### ISSUE TITLE
Support Python f-string

### ISSUE THREAD

**Issue by Jeromearsene** [barklan/inline_sql_syntax#69]
It's impossible to parse [Python f-strings](https://realpython.com/python-f-strings/#python-f-strings-the-pesky-details) for the moment:

✅
![Capture d’écran 2022-06-23 à 09 42 43](https://user-images.githubusercontent.com/9333686/175244350-c8ee607a-0096-42bb-869e-42bd84d8d8b7.png)

❌
![Capture d’écran 2022-06-23 à 09 43 15](https://user-images.githubusercontent.com/9333686/175244378-b01927f1-ff00-4a64-98ca-0da00ca2f1b6.png)

It would be very practical to add this possibility 🙏

**Comment by barklan** [comment]
Added syntax highlighting for multi-line f-strings in 2.14

**Comment by Jeromearsene** [comment]
Sorry, but I don't understand how it works. I tried with [your Python example](https://github.com/barklan/inline_sql_syntax/blob/main/docs/python.png) but got this result:
![Capture d’écran 2022-06-23 à 15 41 15](https://user-images.githubusercontent.com/9333686/175313353-253cbeff-0947-4f69-821e-dd9799573615.png)

So, I conclude that doesn't work.

You explain [here](https://github.com/barklan/inline_sql_syntax#safety) never to do that. Ok, this is not a good practice. But should we understand that it is impossible with your lib ?

Thanks for explanations

**Comment by barklan** [comment]
> But should we understand that it is impossible with your lib ?

No, it's not my place to "punish" people for something that might be unsafe. And there is no indication of unsanitized input here. 
The last two strings in your screenshot work for me. Not sure yet why they don't work for you.

![Screenshot_20220623_184006](https://user-images.githubusercontent.com/71007493/175340791-777456c8-5c11-4305-8282-c9ccaa365d29.png)

I also prematurely closed it without adding support for single-line f-strings) Will add later.

The only string that does not work by design (so far) is this one:

```py
multiline_without_marker = f"""
    SELECT * FROM {table_name};
"""
```

I've open separate issue for queries like this one: #78. Thanks for your input.

**Comment by Jeromearsene** [comment]
With your last version it's good !
![Capture d’écran 2022-06-24 à 10 11 08](https://user-images.githubusercontent.com/9333686/175493068-4eb6ad6f-6b7f-46de-872f-5aa2a4517785.png)

Before update:
![Capture d’écran 2022-06-24 à 10 08 07](https://user-images.githubusercontent.com/9333686/175492614-e7b3066e-9d08-47bc-a44e-fae65c986852.png)

**Comment by torzsmokus** [comment]
> I also prematurely closed it without adding support for single-line f-strings) Will add later.

As far as I can tell, single-line f-strings are still not supported :(
